hack/stop_container.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$@
```

Comment:
This is need to be able to stop process (`mdbook serve --open` process) running within mdbook container, from a shell. Otherwise, user will need to open another shell to kill the running container, delete it. I'm not sure if there is a better solution to this.